android/src/main/java/org/mozilla/mozstumbler/client/mapview/tiles/CoverageOverlay.java

```diff
@@ -16,18 +16,27 @@
  * This class provides the Mozilla Coverage overlay
  */
 public class CoverageOverlay extends AbstractMapOverlay {
+    public static final String MLS_MAP_TILE_COVERAGE_NAME = "Mozilla Location Service Coverage Map";
     // Use a lower zoom than the LowResMapOverlay, the coverage can be very low detail and still look ok
-    private static final int LOW_ZOOM_LEVEL = 10;
+    public static final int LOW_ZOOM_LEVEL = 10;
+
+    public CoverageOverlay(final Context aContext, final String coverageUrl, MapView mapView) {
+        super(aContext);
+        setup(1, AbstractMapOverlay.LARGE_SCREEN_MIN_ZOOM, coverageUrl, mapView);
```

Comment:
I vote yes, since when I look at in future, I'll know exactly for what zoom levels this is active from those values. Arguably, that is also in MapFragment, but that info is hard to see at a glance there.